web/src/containers/admin/EditAccount.test.js

```diff
@@ -0,0 +1,188 @@
+import { shallow } from 'enzyme';
```

Comment:
There aren't any snapshot tests in here.  I think I'm pleased with that, but I'm curious what you think @hbillings?

Reply:
I've never _really_ understood what we get from snapshot tests, so I think I'm fine leaving them out.